Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1542,6 +1543,13 @@ float CClientVehicle::GetHeliRotorSpeed()
     return m_fHeliRotorSpeed;
 }
 
+float CClientVehicle::GetPlaneRotorSpeed()
+{
+    if (m_pVehicle && m_eVehicleType == CLIENTVEHICLE_PLANE)
+        return m_pVehicle->GetPlaneRotorSpeed();
+    return m_fPlaneRotorSpeed;
```

Comment:
I think it looks a bit prettier, and more consistent with surrounding code, if you add a white line before the return:

```suggestion
    if (m_pVehicle && m_eVehicleType == CLIENTVEHICLE_PLANE)
        return m_pVehicle->GetPlaneRotorSpeed();

    return m_fPlaneRotorSpeed;
```